pkg/server/endpoints/eventTracker_test.go

```diff
@@ -0,0 +1,1115 @@
+package endpoints_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/spiffe/spire/pkg/server/endpoints"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPollPeriods(t *testing.T) {
+	for _, tt := range []struct {
+		name         string
+		pollInterval time.Duration
+		pollDuration time.Duration
+
+		expectedPollPeriods uint
+	}{
+		{
+			name:         "polling always polls at least once, even for zero duration",
+			pollInterval: time.Duration(1) * time.Minute,
```

Comment:
nit: all of these time.Duration type coercions aren't needed and IMO hurt readability

Reply:
Sorry, the default time code uses this pattern in specifying Durations.  I thought, it being in the examples of "time" package, it was the standard way of representing one minute / second / etc.

I'll scrub all `time.Duration(1)` elements from the submit.